eva-accession-release/src/main/java/uk/ac/ebi/eva/accession/release/listeners/ExcludeVariantsListener.java

```diff
@@ -31,6 +31,8 @@
     @Override
     public ExitStatus afterStep(StepExecution stepExecution) {
         logger.info("Processors filtered out {} variants", stepExecution.getFilterCount());
+        logger.warn("Processors skipped {} variants because the start position is greater than the chromosome end",
+                    stepExecution.getSkipCount());
```

Comment:
if we are not writing the list of contigs affected here, we don't need the variantsWithIllegalStart set in the processor.